### DISCUSSION TITLE
Awesome book!

### DISCUSSION THREAD

**Discussion by johnoestmannmusic** [csound-flossmanual/csound-floss#96] [General]
Hey team,

I am relatively early in the book, but I had to take a moment to say thanks for all the work the team has put in. I'm a game composer with some experience with Max, PD, SuperCollider, UE Metasounds, others, and I have to say from what I've worked through so far, this is one of the clearest introductions to a computer music language I've experienced. Really appreciate the extra depth at the end of each chapter too, as it's probably the clearest way I've seen dB in real world vs digital audio explained (as one example).

Much appreciated!

John

**Comment by joachimheintz** [comment]
thanks very much john -

it is really motivating to know that you find this useful!  (motivating 
to improve it - as far as time and energy allows it.  it is a continuous 
process, and a challenge once csound 7 is out.)

any suggestions from your side are welcome, in particular as you have so 
much experience with other programming languages.

   joachim
